base/src/main/java/com/maubis/scarlet/base/core/format/FormatType.kt

```diff
@@ -13,5 +13,5 @@ enum class FormatType {
   CODE,
   QUOTE,
   SEPARATOR,
-  EMPTY,
+  EMPTY
```

Comment:
Undo this maybe?  

Format should ideally be 1:1 to actual UI blocks. :)

Reply:
Hmm what's the problem here? I've just removed a trailing comma, the actual content hasn't changed.

Reply:
Oh sorry. That was a stupid comment.